crates/hyperqueue/src/dashboard/ui/screens/home/chart_section.rs

```diff
@@ -0,0 +1,114 @@
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+use tako::messages::gateway::CollectedOverview;
+
+use crate::dashboard::ui::terminal::DashboardFrame;
+use std::cmp::Ordering::Equal;
+use tui::layout::{Alignment, Rect};
+use tui::style::{Color, Modifier, Style};
+use tui::symbols;
+use tui::text::{Span, Spans};
+use tui::widgets::{Axis, Block, Borders, Chart, Dataset, Paragraph, Wrap};
+
+#[derive(Clone)]
+pub struct ClusterOverviewChart {
+    /// (time, #workers)
+    worker_count_vec: Vec<(f64, f64)>,
```

Comment:
I would keep some struct here. If you can encode something using the type system, it's better than putting it into comments. There's no reason not to create a struct here.